### PR TITLE
Last minute bugs

### PR DESCRIPTION
**Pull request by LucaHaverty**
### Description
Some additional small bugs that were found on 8/23

### Bugs
- [x] On scoring zone panel, edit -> cancel -> edit caused a null ref exception
- [x] Exiting to the main menu after using the move tool caused a null ref exception
- [x] Robot build delete part/robot accept button color changes when hovered over
- [x] Main HUD hover tint effects stopped working after a merge
- [x] Multibot -> Add allows users to spawn a Mix and match robot
- [x] The gizmo pointer object was being created before the previous one was destroyed, causing issues when chaining gizmo calls together in the robot builder

Anyone is welcome to add small fixes to this PR

### CODE REVIEW

**Review by Markytt — APPROVED** [review]
Good fixes

**Review by a-crowell — APPROVED** [review]
Works well, nice job!

File: engine/Assets/Scripts/UI/Dynamic/Panels/Configuring/Scoring/ZoneConfigPanel.cs [hunk 1]
```diff
@@ -60,7 +60,18 @@ public class ZoneConfigPanel : PanelDynamic {
             _isNewZone            = false;
             _initialData.Name     = zone.Name;
             _initialData.Alliance = zone.Alliance;
-            var parent = zone.GameObject.transform.parent.name == "grounded" ? null : zone.GameObject.transform.parent;
+
+            /*Transform parent;
+            if (zone.GameObject.transform.parent == null)
+                parent = null;
+            else parent = zone.GameObject.transform.parent.name == "grounded" ? null :
+            zone.GameObject.transform.parent;*/
```
Comment (BrandonPacewic) [inline engine/Assets/Scripts/UI/Dynamic/Panels/Configuring/Scoring/ZoneConfigPanel.cs:68]:
```suggestion
```

Can be removed?

**Review by BrandonPacewic — APPROVED** [review]

**Comment by KyroVibe** [comment]
@BrandonPacewic I made a quick tweak to remove the error when analytics isn't setup

**Review by BrandonPacewic — APPROVED** [review]
> @BrandonPacewic I made a quick tweak to remove the error when analytics isn't setup

Still looks good 👍

**Comment by LucaHaverty** [comment]
> I made a quick tweak to remove the error when analytics isn't setup

Awesome, thanks!

**Pull request merged**